javascript/mcp-server/src/index.ts

```diff
@@ -244,7 +309,25 @@ async function main() {
   await server.connect(transport)
 }
 
-main().catch((error) => {
-  console.error('Fatal error in main():', error)
-  process.exit(1)
-})
+function main() {
+  const program = new Command()
+
+  program.name('forevervm-mcp').version('1.0.0')
+
+  program
+    .command('install')
+    .description('Set up the ForeverVM MCP server')
+    .option('-c, --claude', 'Set up the MCP Server for Claude Desktop')
+    .action(installForeverVM)
+
+  program.parse(process.argv)
+
+  if (!process.argv.slice(2).length) {
+    console.log('Starting ForeverVM MCP Server...')
+    runMCPServer().catch((error) => {
+      throw Error('Failed to start MCP Server')
+    })
+  }
```

Comment:
Thoughts on just making a subcommand called `run`? Now that we have an installer, it doesn't add any burden on our users; we can just have the installer add `run` to the command args when it installs.

```suggestion
  program
    .command('run')
    .description("Run the MCP server (you shouldn't need to run this; your client will)")
    .action(runMCPServer)

  // optionally, we could make run the default like this:
  // if (process.argv.length === 2) {
  //   process.argv.push('run')
  //   return
  // }
  
  program.parse(process.argv)
```

(surprisingly, this seems to be the best way to do it with commander.js)

Reply:
i thought about it, but i don't think it really matters, since users don't run the command, it just gets pasted in the config file

Reply:
I'm mainly thinking that if someone just runs `npx forevervm-mcp` and we default to running the MCP server, they could be confused by what they're supposed to do, whereas if we move it behind `run`, the default behavior will be to print a help message that lists commands. I think that's the better UX in that case.